.ci/pipelines/e2e-tests-ocp-all-but-latest.Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+                    // it requires
+                    isFriday()
+               }
+            }
+            steps {
+                // "4.3.40", "4.4.33", "4.5.37", "4.6.24", "4.7.6"
+                // 4.8.5 is taken care of by a separate job
+                build job: 'cloud-on-k8s-e2e-tests-ocp',
+                                    parameters: [
+                                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: JKS_PARAM_OPERATOR_IMAGE),
+                                        string(name: 'OCP_VERSION', value: "4.3.40"),
+                                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                                    ],
+                                    wait: true
+                build job: 'cloud-on-k8s-e2e-tests-ocp',
+                                    parameters: [
+                                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: JKS_PARAM_OPERATOR_IMAGE),
+                                        string(name: 'OCP_VERSION', value: "4.4.33"),
+                                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                                    ],
+                                    wait: true
+                build job: 'cloud-on-k8s-e2e-tests-ocp',
+                                    parameters: [
+                                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: JKS_PARAM_OPERATOR_IMAGE),
+                                        string(name: 'OCP_VERSION', value: "4.5.37"),
+                                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                                    ],
+                                    wait: true
+                build job: 'cloud-on-k8s-e2e-tests-ocp',
+                                    parameters: [
+                                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: JKS_PARAM_OPERATOR_IMAGE),
+                                        string(name: 'OCP_VERSION', value: "4.6.24"),
+                                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                                    ],
+                                    wait: true
+                build job: 'cloud-on-k8s-e2e-tests-ocp',
+                                    parameters: [
+                                        string(name: 'JKS_PARAM_OPERATOR_IMAGE', value: JKS_PARAM_OPERATOR_IMAGE),
+                                        string(name: 'OCP_VERSION', value: "4.7.6"),
+                                        string(name: 'branch_specifier', value: GIT_COMMIT)
+                                    ],
+                                    wait: true
+            }
+        }
+    }
+}
+
+def isFriday() {
+    // %u day of week (1..7); 1 is Monday 5 is Friday
+    return sh (
+        script: "date +%u",
+        returnStdout: true
+    ) as Integer == 5
+}
```

Comment:
Add a new empty line.

Reply:
I think there is already one, otherwise we would have the funny little red indicator?

Reply:
It was [there](https://github.com/elastic/cloud-on-k8s/pull/4754/commits/941fbd370fe3de10b9c2de764f349f1c568992c2) but you [fixed](https://github.com/elastic/cloud-on-k8s/pull/4754/commits/a72cea826544de251cca4aaab4e84556b3c80ccf) it.

Reply:
Totally missed that, sorry!